src/portal/lib/src/tag/tag-detail.component.spec.ts

```diff
@@ -21,7 +21,7 @@ import {
   ScanningResultDefaultService
 } from "../service/index";
 import { FilterComponent } from "../filter/index";
-import { VULNERABILITY_SCAN_STATUS } from "../utils";
```

Comment:
Don't forget to add more test case for your component to make sure branch coverage for it 40%+.

Reply:
Do you mean statements coverage? Current Target: global: {
            statements: 40,
            branches: 13,
            functions: 26,
            lines: 41
          },